src/components/container/BasicPointsForBadges/BasicPointsForBadges.tsx

```diff
@@ -78,7 +78,7 @@ export default function (props: BasicPointsForBadgesProps) {
     return <div className={"mdhui-basic-points-for-badges"} ref={props.innerRef}>
         {badges !== undefined && points !== undefined &&
             <>
-                <Title order={1} className="mdhui-basic-points-for-badges-points-toward-badge" style={{ color: resolveColor(layoutContext.colorScheme, props.pointsLabelColor) }}>{props.showTotalPoints ? points.toLocaleString() : (props.pointsPerBadge - pointsUntilNextBadge()).toLocaleString()}pts</Title>
+                <Title order={1} className="mdhui-basic-points-for-badges-points-toward-badge" color={resolveColor(layoutContext.colorScheme, props.pointsLabelColor)} >{props.showTotalPoints ? points.toLocaleString() : (props.pointsPerBadge - pointsUntilNextBadge()).toLocaleString()}pts</Title>
```

Comment:
You can just use `color={props.pointsLabelColor}` here.  The resolution of the color already occurs within the `Title` component.

Same thing below for the progress bar fill color.

Once those are updated, you can probably remove the `layoutContext` all together, as I don't think it is used anywhere else.